com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/PathTracing.cs

```diff
@@ -536,6 +636,33 @@ TextureHandle RenderPathTracing(RenderGraph renderGraph, HDCamera hdCamera, Text
         {
             m_PathTracingSettings = hdCamera.volumeStack.GetComponent<PathTracing>();
 
+            var motionVector = TextureHandle.nullHandle;
+            var albedo = TextureHandle.nullHandle;
+            var normal = TextureHandle.nullHandle;
+
```

Comment:
Hmm, I don't see how (at least without rewriting parts of the code).

The idea is that these variables are always passed to the RenderPathTracingFrame functions, but should remain null if we don't need AOVs or if we don't have the denoising plugin at all.

Is there are more clean (or readable) way to do it?

Reply:
I mean, right after the "// Check the validity of the state before moving on with the computation" test, which returns right away.

Reply:
Oh yes, you are right, the validity test should be first. I was confused because I though "the following block" was referring to only the 3 lines that github shows in the preview. I fixed it now!